src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -129,7 +129,6 @@
     <PlatformManifestFileEntry Include="mscordbi.dll" IsNative="true" />
     <PlatformManifestFileEntry Include="libmscordbi.so" IsNative="true" />
     <PlatformManifestFileEntry Include="libmscordbi.dylib" IsNative="true" />
-    <PlatformManifestFileEntry Include="mscorrc.dll" IsNative="true" />
```

Comment:
I think it may need to stay here. I believe that this file needs to include all files that ever shipped in the platform, even the ones that are no longer shipping.